django-rgd/client/rgd_client/plugin.py

```diff
@@ -13,6 +14,49 @@ class RgdPlugin:
     def __init__(self, session: RgdClientSession):
         self.session = session
 
+    def download_checksum_file_to_path(
+        self, id: int, path: Optional[Path] = None, keep_existing=False, use_id=False
+    ):
+        """
+        Download a RGD ChecksumFile to a given path.
+
+        Args:
+            id: The id of the RGD ChecksumFile to download.
+            path: The root path to download this file to.
+            keep_existing: If False, replace files existing on disk.
```

Comment:
```suggestion
            keep_existing: If False, replace files existing on disk.
            use_id: If True, save this file to disk using it's ID, rather than it's name.
```